src/collection.js

```diff
@@ -515,19 +515,17 @@ export default class Collection {
    * @return {Promise}
    */
   _updateRaw(oldRecord, newRecord, {synced = false} = {}) {
-    let updated;
     // Make sure to never loose the existing timestamp.
     if (oldRecord && oldRecord.last_modified && !newRecord.last_modified) {
       newRecord.last_modified = oldRecord.last_modified;
     }
     // If only local fields have changed, then keep record as synced.
     const isIdentical = oldRecord && recordsEqual(oldRecord, newRecord, this.localFields);
     const keepSynced = isIdentical && oldRecord._status == "synced";
-    let newStatus = (keepSynced || synced) ? "synced" : "updated";
-    if (!oldRecord) {
-      newStatus = "created";
-    }
-    updated = markStatus(newRecord, newStatus);
+    const neverSynced = oldRecord && oldRecord._status == "created";
+    const newStatus = (keepSynced || synced) ? "synced"
+                                             : neverSynced ? "created" : "updated";
+    const updated = markStatus(newRecord, newStatus);
```

Comment:
Also, I guess I just thought of another situation where we want to change it, and that's if we're updating a "deleted" record -- it should become "updated" instead.